procedures/unit-testing-basics.ipf

```diff
@@ -1313,12 +1313,30 @@ static Function/S getTestCaseList(procWin)
 	testCaseMDList = GrepList(testCaseMDList, PROCNAME_NOT_REENTRY)
 
 	if(!UTF_Utils#IsEmpty(testCaseMDList))
-		testCaseList = testCaseList + testCaseMDList
+		testCaseList = SortTestCaseList(procWin, testCaseList + testCaseMDList)
```

Comment:
Here you rely on the fact that for the case of testCaseMDList == "" the FunctionList functions returns the function names in the order they appear in the procedure file. However that is not specified by FunctionList.
So it would be correct to apply SortTestCaseList also in that case.